documentation/PYTHON.md

```diff
@@ -1,87 +1,150 @@
 # Python Interface {#python_interface}
 
-In the following we will give a detailed overview how to specify models in  Python and how to call the generated simulation files.
+In the following we will give a detailed overview how to specify models in
+Python and how to call the generated simulation files.
 
 ## Model Definition
 
-This guide will guide the user on how to specify models in Python using SBML. For example implementations see the examples in the python/examples directory.
+This guide will guide the user on how to specify models to import and simulate
+them using the Python interface.
+For example implementations see the examples in the `python/examples`
+directory.
 
-### SBML input
+### SBML import
 
-First, import an sbml file using the `amici.sbml_import.SbmlImporter` class:
+[SBML](http://sbml.org/) is a commonly used format for specifying systems
```

Comment:
As this section involves code, doesn't it make more sense to also move this to an example notebook?

Reply:
same applies to later parts of this documentation.

Reply:
Had the same feeling. But didn't want to just remove it. I think we want to have the notebooks rendered into the sphinx documentation in the future. Once they are in the proper places, I'd drop these code examples. Would leave it until then...